### PR TITLE
Improve Layout of committee detail view

### LINKED ISSUE #1574: Improve Layout of committee detail view
The lists of committees motions can be forwarded to and received from should be collapsible if there is more than one committee in the list.
For users without access to the listed committee only the name should be displayed (not as link).

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #1574

### CODE REVIEW

File: client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-view/components/committee-detail-view/committee-detail-view.component.html [hunk 1]
```diff
@@ -41,13 +41,18 @@ <h1>
             [title]="forwardLabel"
             *ngIf="committee.forward_to_committees.length > 0"
         >
-            <ul class="content-list">
+            <ul class="content-list expandable-list" [ngClass]="forwardingExpanded ? 'expanded' : ''">
```
Comment (tsiegleauq) [inline client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-view/components/committee-detail-view/committee-detail-view.component.html:44]: `[class.expanded]="forwardingExpanded"`

File: client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-view/components/committee-detail-view/committee-detail-view.component.html [hunk 2]
```diff
@@ -56,13 +61,18 @@ <h1>
             [title]="receiveLabel"
             *ngIf="committee.receive_forwardings_from_committees.length > 0"
         >
-            <ul class="content-list">
+            <ul class="content-list expandable-list" [ngClass]="receiveExpanded ? 'expanded' : ''">
```
Comment (tsiegleauq) [inline client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-view/components/committee-detail-view/committee-detail-view.component.html:64]: `[class.expanded]="receiveExpanded"`

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
Only the orga admin should see the "Forward motions to"/"Receive motions from" fields because committee admins have not read access for all committees (-> they get an incomplete commitee list which should be prevent).
So please add a new check that orga admins can see only the fields.

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
Please order the committes by name (currently it looks like by id which is not helpful)

**Review by emanuelschuetze — APPROVED** [review]

**Review by rrenkert — APPROVED** [review]

**Pull request merged**